google/resource_google_project.go

```diff
@@ -215,20 +215,27 @@ func prefixedProject(pid string) string {
 }
 
 func getParentResourceId(d *schema.ResourceData, p *cloudresourcemanager.Project) error {
-	if v, ok := d.GetOk("org_id"); ok {
-		org_id := v.(string)
+	orgId, orgOk := d.GetOk("org_id")
+	folderId, folderOk := d.GetOk("folder_id")
+
+	if orgOk && folderOk && orgId != "" && folderId != "" {
```

Comment:
nit: the ok just means "is not the zero value for this type", so `orgOk` will always be the same as `orgId != ""`

Reply:
Done